lambda/helper/python/comprehendHelper.py

```diff
@@ -443,10 +449,12 @@ def processComprehend(self,
                                        documentPath)
                                   
         # process comprehend medical data, create the entities result file in S3
-        self.processComprehendMedicalEntities(comprehendMedicalEntities,
+        comprehendMedicalEntities = self.processComprehendMedicalEntities(comprehendMedicalEntities,
```

Comment:
Now that the function is doing 2 tasks,should we consider renaming the function as processAndReturnComprehendMedicalEntities? I'd like to have the function names as clear as possible

Reply:
Not a bad idea. Will do.